hil/ext/switches/mock.py

```diff
@@ -56,6 +56,11 @@ def validate(kwargs):
             'password': basestring,
         }).validate(kwargs)
 
+    @staticmethod
+    def validate_port_name(port):
+        """All port names are valid for the mock switch"""
+        pass
```

Comment:
The idea of mock switch is that people can simulate the behavior that happens in production system.
I think, here also you should put the port validation code as used in the dell switch. 
For clarity, just mention in the docstring an example of the correct format for port number to be passed for the mock switch.